public/index.html

```diff
@@ -2,44 +2,99 @@
 <html>
 <head>
 	<title>Ping Pong Game</title>
+	<link rel="stylesheet" href="css/app.css">
 </head>
 <body>
-	<canvas id="canvas" width="800" height="400"></canvas>
+	<canvas id="canvas"></canvas>
 	<script type="text/javascript" src="js/app.js"></script>
+	<script type="text/javascript" src="js/resize.js"></script>
+	<script type="text/javascript" src="js/pad.js"></script>
+	<script type="text/javascript" src="js/ball.js"></script>
 	<script type="text/javascript">
-		//Sample functions, remove these functions and design ping-pong game with onInit and onUpdate.
+		app.restartMatch = function() {
+			ball.resetPosition();
+			player1.resetPosition();
+			player2.resetPosition();
+		}
 
 		app.onInit = function(){
+			app.reset();
+			app.initialised = true;
+		};
+
+		// pause toggle function
+		app.togglePause = function() {
+			app.paused = !app.paused;
+			console.log(`Game paused: ${app.paused}`);
+		}
+
+		// reset function
+		app.reset = function() {
+			this.nodes = [];
+
+			// add nodes
 			this.nodes.push({
-				id : 'red-box',
-				x  : 100,
+				id : 'midline',
+				x  : app.width / 2 - 10,
 				y  : 0,
-				width  : 100,
-				height : 100,
-				color  : 'red',
-				direction : 0
+				width  : 20,
+				height : app.height,
+				color  : 'black',
 			});
-
 			this.nodes.push({
-				id : 'black-box',
-				x  : 50,
-				y  : 0,
-				width  : 150,
-				height : 150,
-				color  : 'black'
+				id : 'top',
+				x  : 0,
+				y  : -20,
+				width  : app.width,
+				height : 20,
+				color  : 'black',
 			});
-		};
+			this.nodes.push({
+				id : 'bottom',
+				x  : 0,
+				y  : app.height,
+				width  : app.width,
+				height : 20,
+				color  : 'black',
+			});
+
+			player1 = new Pad('player1', 10, app.height / 2 - 50, 'w', 's');
+			player2 = new Pad('player2', app.width - 30, app.height / 2 - 50, 'ArrowUp', 'ArrowDown');
+			ball = new Ball('ball', app.width / 2 - 5, app.height / 2 - 5);
```

Comment:
5. Can you make sure sizes are according to appsize and not static

Reply:
I've done some refactoring there.

But I designed the game so the game area is a fixed size, while the camera viewport fits the area to the screen.
So I'm not sure how changing those numbers would make sense?